.github/README.md

```diff
@@ -45,10 +45,9 @@ The Oppia code is released under the [Apache v2 license](https://github.com/oppi
 
 ## Keeping in touch
 
-  * [Discussion forum](http://groups.google.com/group/oppia)
-  * [Announcements mailing list](http://groups.google.com/group/oppia-announce)
+  * [Discussion forum](https://github.com/oppia/oppia/discussions)
+  * [GitHub Discussions](https://github.com/oppia/oppia/discussions)
```

Comment:
Er, no -- keep the announcements list, swap out the other one.

Reply:
Also you missed the change on line 12.

Reply:
Yes Done.